.env.example

```diff
@@ -4,8 +4,13 @@ PORT=3000
 # Systemd service to restart if the check fails
 SYSTEMD_SERVICE="proxied_rpc.service"
```

Comment:
PS. Maybe this should be removed?

Reply:
Probably it can yes, but I wanted to keep it for a few days using ethers, just as a fail safe. Until we check how well IPC works